package.json

```diff
@@ -29,23 +29,21 @@
   },
   "devDependencies": {
     "@changesets/cli": "^2.26.1",
-    "@remix-run/node": "^1.12.0",
-    "@remix-run/react": "^1.12.0",
+    "@remix-run/node": "^1.17.0",
+    "@remix-run/react": "^1.17.0",
     "@tailwindcss/typography": "^0.5.9",
-    "@types/jest": "^29.4.1",
-    "@types/react": "^18.0.26",
-    "@types/react-dom": "^18.0.10",
+    "@types/react": "^18.2.12",
+    "@types/react-dom": "^18.2.5",
+    "concurrently": "^8.2.0",
     "eslint-config-curvenote": "latest",
-    "glob": "^10.2.7",
```

Comment:
@stevejpurves I had to delete this to get storybook to work. The copy script still seems to work!